src/snippets/extending-classes.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Extending classes, object orientated programming snippet
+---
+
+# Extending Classes
+
+Extending classes, and general Object Orientated Programming (OOP) in AssemblyScript
+
+
+```editor
+#!optimize=size&runtime=full
```

Comment:
```suggestion
#!runtime=full
```

Whether the full runtime is good here or not again depends a bit. With it, the wat file is considerably larger, but as we are dealing with classes here as well, one gets the exported runtime helpers which might be useful. Perhaps a `string` case in the HTML?

Reply:
Same as above :smile: :+1: